prow/jobs/images/Dockerfile.unit

```diff
@@ -0,0 +1 @@
+FROM prow/test as base
```

Comment:
nit: we can also tag `prow/test` image with `prow/unit` - it is safer and will prevent from using out dated images or unwanted pulls.